client/app/hosting/general-informations/GENERAL_INFORMATIONS.controller.js

```diff
@@ -1,18 +1,32 @@
 angular
     .module("App")
     .controller("hostingGeneralInformationsCtrl", class HostingGeneralInformationsCtrl {
-        constructor ($scope, $stateParams, Alerter, hostingSSLCertificate, translator) {
+        constructor ($scope, $stateParams, Alerter, hostingSSLCertificate, HostingRuntimes, translator) {
             this.$scope = $scope;
             this.$stateParams = $stateParams;
 
             this.Alerter = Alerter;
             this.hostingSSLCertificate = hostingSSLCertificate;
             this.translator = translator;
+            this.HostingRuntimes = HostingRuntimes;
         }
 
         $onInit () {
             this.$scope.$on("hosting.ssl.reload", () => this.retrievingSSLCertificate());
 
+            this.loading = {
+                defaultRuntime: true
+            };
+
+            this.defaultRuntime = null;
+            this.HostingRuntimes.getDefault(this.$stateParams.productId)
+                .then((runtime) => {
+                    this.defaultRuntime = runtime;
+                })
```

Comment:
No catching ?

Reply:
`finally` does the trick

Reply:
Act more like displaying a message to the user that something went wrong.

Reply:
It actually does. Following @cbourgois's advice, another span shows'up in red when something goes wrong.